taskcluster/kinds/evaluate-quantized/kind.yml

```diff
@@ -99,7 +99,8 @@ tasks:
                     --trg               {trg_locale}
                     --models            "$MOZ_FETCHES_DIR/model.intgemm.alphas.bin"
                     --dataset_prefix    "$MOZ_FETCHES_DIR/{dataset_sanitized}"
-                    --vocab             "$MOZ_FETCHES_DIR/vocab.spm"
+                    --vocab_src          "$MOZ_FETCHES_DIR/vocab.{src_locale}.spm"
+                    --vocab_trg          "$MOZ_FETCHES_DIR/vocab.{trg_locale}.spm"
```

Comment:
Just a heads up. OpusMT seems to call these just `source.spm` and `target.spm`. I'm not suggesting that OpusMT is an authority on this, but it is important to note that these names are assumed by HuggingFace's MarianTokenizer:

https://github.com/huggingface/transformers/blob/main/src/transformers/models/marian/tokenization_marian.py#L30

Perhaps for future compatibility it is best to follow these names?

Reply:
I'd prefer the current naming, as it matches our ecosystem. Our models don't work outside of our ecosystem currently. We can always re-export things as needed once we do support running our models outside of our ecosystem.